src/main/java/org/purpurmc/purpur/client/gui/SplashTexture.java

```diff
@@ -1,13 +1,16 @@
 package org.purpurmc.purpur.client.gui;
 
-import org.purpurmc.purpur.client.PurpurClient;
 import com.mojang.blaze3d.platform.NativeImage;
-import java.io.IOException;
-import java.io.InputStream;
 import net.minecraft.client.renderer.texture.SimpleTexture;
+import net.minecraft.client.renderer.texture.TextureContents;
 import net.minecraft.client.resources.metadata.texture.TextureMetadataSection;
 import net.minecraft.resources.ResourceLocation;
 import net.minecraft.server.packs.resources.ResourceManager;
+import org.jetbrains.annotations.NotNull;
+import org.purpurmc.purpur.client.PurpurClient;
+
+import java.io.IOException;
+import java.io.InputStream;
```

Comment:
unnecessary import changes

Reply:
How do I disable it in IntelliJ just for that file? In MixinLoadingOverlay it is ok to have it seperated but not here?